### ISSUE TITLE
Add option to hide values to Html defaults

### ISSUE THREAD

**Issue by wykys** [openscopeproject/InteractiveHtmlBom#324]
It can look messy on some boards if lots of values are displayed on top of each other. It is possible to hide or align the values, but in many cases it is unnecessary work. This option could increase clarity.

**Comment by qu1ck** [comment]
Why not hide fab layer then? (I'm assuming the mess you are talking about is not on silk).

**Comment by wykys** [comment]
Because in the fab layer there are not only values, but also references or more detailed details of components.

I'm looking for an easy way to hide values.

**Comment by qu1ck** [comment]
Fair enough. I'll add this but just as a reminder, it's very easy to batch hide all values on specific layer in kicad using "Edit text and graphic properties" dialog.
